.github/workflows/tests.yaml

```diff
@@ -168,6 +168,8 @@ jobs:
   # Publish an artifact for the event; used by publish-test-results.yaml
   event_file:
     # Do not run the schedule job on forks
+    permissions:
+      contents: none
```

Comment:
This appears to be a bit random. We're using the upload-artifact three times but this is the only one where you suggest to limit permissions. Why is that?